gremlin/values.yaml

```diff
@@ -81,38 +81,21 @@ gremlin:
   container:
     # gremlin.container.driver -
     # Specifies which container driver with which to run Gremlin. Possible values and their meanings:
-    #   docker           =   Provides legacy integration with Docker containers
```

Comment:
`docker` is still in the dictionary in the helpers template, and container drivers list

Reply:
Good catch